flink/v1.16/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergFilesCommitterMetrics.java

```diff
@@ -37,6 +40,9 @@ class IcebergFilesCommitterMetrics {
         metrics.addGroup("IcebergFilesCommitter").addGroup("table", fullTableName);
     committerMetrics.gauge("lastCheckpointDurationMs", lastCheckpointDurationMs::get);
     committerMetrics.gauge("lastCommitDurationMs", lastCommitDurationMs::get);
+    this.elapsedSecondsSinceLastSuccessfulCommit = new ElapsedTimeGauge(TimeUnit.SECONDS);
```

Comment:
Iceberg commit happens after a successful checkpoint. Iceberg commits can fail (for whatever reason) while Flink checkpoints succeeded. this metric is very useful to set up alerts to detect consecutive Iceberg commit failures.

Will send a separate PR for doc on FlinkSink metrics.

Reply:
PR #6765 for doc update